dicoogle/src/main/resources/webapp/js/components/search/result/studyView.js

```diff
@@ -102,7 +102,8 @@ const StudyView = createReactClass({
               uid: item.studyInstanceUID,
               // deprecated data fields
               "data-result-type": "study",
-              "data-result-uid": item.studyInstanceUID
+              "data-result-uid": item.studyInstanceUID,
+              "data-result-patientid": item.patientid
```

Comment:
With this reasoning, maybe we should also provide more ID fields at the other levels?